csfieldguide/static/interactives/cfg-equation-builder/scss/cfg-equation-builder.scss

```diff
@@ -0,0 +1,59 @@
+#txt-description {
```

Comment:
Could probably use the bootstrap class `text-center` instead of defining a new class.

Same goes for the `#txt-try` and `#error-notice` - could remove the text alignment rule and use `text-center`.